classes/01class/exercises/c01-git01/frdvo/ANSWER.md

```diff
@@ -0,0 +1,36 @@
+# c01-git01
```

Comment:
This file should not be here.. :) 
I guess you've create this new branch from your previous branch(frdvo/c01-git01) insteaf od master..

Reply:
good call... I have to `git checkout master` before starting the next exercise! my mistake. Thanks for the explanation.